packages/thirdweb/src/react/core/hooks/contract/useSendTransaction.ts

```diff
@@ -20,7 +21,7 @@ type ShowModalData = {
   tx: PreparedTransaction;
   sendTx: () => void;
   rejectTx: () => void;
-  totalCostWei: bigint;
+  cost: string;
```

Comment:
what's the unit here? also do we still need this?

Reply:
unit is number of tokens. This object is only used internally - this is not public API